src/Services/SchemaPoster.php

```diff
@@ -98,18 +95,49 @@ public function postSchema(string $server_id): array {
       ])
             ? 'info'
             : 'error';
-    $this->logger->{$log_function}('Files uploaded: {status_code} {reason}', [
-          'status_code' => $response->getStatusCode(),
-          'reason' => $response->getReasonPhrase(),
-      ]);
-    $message = vsprintf($this->t('Result: %s Status code: %d - %s'), [
-          $log_function == 'error' ? 'NOT UPLOADED' : 'UPLOADED',
-          $response->getStatusCode(),
-          $response->getReasonPhrase(),
-      ]);
+      $this->logger->{$log_function}('Files uploaded: {status_code} {reason}', [
+            'status_code' => $response->getStatusCode(),
+            'reason' => $response->getReasonPhrase(),
+        ]);
+      $message = vsprintf($this->t('Result: %s Status code: %d - %s'), [
+            $log_function == 'error' ? 'NOT UPLOADED' : 'UPLOADED',
+            $response->getStatusCode(),
+            $response->getReasonPhrase(),
+        ]);
+    }
     return [$message];
   }
 
+  /**
+   * @throws \ZookeeperConnectionException
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \ZookeeperException
+   * @throws \Drupal\search_api\SearchApiException
+   * @throws \Drupal\search_api_solr\SearchApiSolrException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   */
+  protected function updateZookeeperConfigs(string $server_id) {
+    $message = '';
+    $schemaFiles = $this->getSolrFiles($server_id);
+    $projectName = getenv('PROJECT_NAME');
+    $zk = new \Zookeeper(getenv('PANTHEON_CONFIGSERVER_HOST') . ":" . getenv('PANTHEON_CONFIGSERVER_PORT'));
```

Comment:
Maybe pass in host and port as parameters, and read the env vars from the caller?